lifterlms/courses-lessons/exclude-posts-from-user-roles.php

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Can we add the PHP line here to make this a valid PHP file? Looks like most snippets have it with a comment to exclude it when copy/pasting into an existing PHP file:

```
<?php // Don't copy this line!
```